docs/explanations/what_is_stardust/what_is_stardust.md

```diff
@@ -16,9 +16,51 @@ Stardust is a comprehensive utility upgrade of the IOTA Chrysalis protocol. The
+- making IOTA an **infrastructure and settlement layer** for [second-layer smart contract chains](https://wiki.iota.org/smart-contracts/overview) using IOTA Smart Contracts (ISC)
+- transforming IOTA into a **multi-asset ledger** with custom, user-defined tokens using the new tokenisation framework.
+Let’s examine each of these separately. 
 
-The Chrysalis ledger was redesigned and extended to cater to these goals. In the following, we will go through the new
-concepts in detail.
+## Tokenization framework
+The tokenization framework enables a lot of new functionality that builders want. Most importantly the ability to mint other tokens and NFTs. Our approach is different to the approach used in most other platforms. The advantages are:
+- Tokenization has low energy requirements and no fees.
+- Minting is fast and easy and can even be done on Firefly!
+- Native tokens and NFTs are first class citizens of the core protocol
+-- They benefit from the security of Layer 1.
+-- It is simple to fetch all tokens belonging to an account (with Ethereum your ERC balance is stored in the contract state making it difficult to quickly get your balance).
+-- Adding native tokens should be extremely easy for exchanges (easier than adding ERC20 tokens for Ethereum).
+- Storage deposits are needed for tokenization, which gives significant utility to the IOTA / Shimmer token. The barrier for minting still remains low as you only need a small amount for the initial mint (with additional storage deposits needed for each subsequent wallet holding that token).
+- Storage deposits allow for permanent storage of data on the Tangle. If permanence of data is deemed sufficiently important for a given use case it is possible to store data indefinitely with a storage deposit that scales linearly with the amount of data being stored. If you are no longer using the storage, you get the deposit back.
+- The storage deposits allow on-chain storage of NFT metadata, so that you are not reliant on an IPFS cluster or a web2 solution being maintained. This is relatively unique for NFTs in the DLT space - you would not store a whole movie, but there is a lot of opportunity for novel use cases, especially about storing important metadata. Alternatively this could be useful for some rare high value NFTs (eg “Ferrari” NFTs) for which one may happily pay for the maximum 32kb storage limit. This would also lock up a larger amount of IOTA tokens, which may be seen as a good thing.
+- Unlock conditions - multiple use cases such as time locks. But an important use case is that no one can spam your address with tokens you don’t want. Eg taint your address with Tornado cash tokens (although in our case it would only taint the utxo it is on rather than the whole address).
+-- Requirements for transaction claiming also apply to NFTs and ensure no one can spam you with unwanted NSFW content (as per https://decrypt.co/79406/budweiser-dick-pic-nft-ethereum-wallet)  
```

Comment:
They can spam you with NSFW NFTs, but it means they are giving you the storage deposit plus you can burn them.

Reply:
Moved to the storage deposit section